pandas/io/tests/test_pickle.py

```diff
@@ -24,8 +24,8 @@ class TestPickle():
 
     1. Install pandas version intended to output the pickle.
 
-    2. Execute "generate_legacy_pkcles.py" to create the pickle.
-    $ python generate_legacy_pickles.py <version> <output_dir>
+    2. Execute "generate_legacy_storage_files.py" to create the pickle.
+    $ python generate_legacy_storage_files.py <version> <output_dir> pickle
```

Comment:
The reason is to generates test methods automatically. Pls refer:

https://github.com/kawochen/pandas/blob/BUG-FIX-9618/pandas/io/tests/test_pickle.py#L32